### PR TITLE
Tyr: add aggregate_places command

### PR DESCRIPTION
**Pull request by l-vincent-l**
Launch aggregate places.
It can take a name argument to specify on which instance we want
to aggregate places.

### CODE REVIEW

File: source/tyr/tyr/aggregate_places.py [hunk 1]
```diff
@@ -148,3 +150,31 @@ class NetworkEd(Base_ed):
...
+
+    def get_options(self):
+        return [
+            Option('-n', '--name', dest='name_', default="",
+                   help="Launch aggregate places, if no name is specified "
+                   "this is launched for all instances")
+        ]
+
+    def run(self, name_=""):
+        if name_ != "":
```
Comment (antoine-de) [inline source/tyr/tyr/aggregate_places.py]: juste if name_ ca suffit pas ?

File: source/tyr/tyr/aggregate_places.py [hunk 2]
```diff
@@ -148,3 +150,31 @@ class NetworkEd(Base_ed):
...
+        else:
+            instances = models.Instance.query.all()
+
+        job_id = 1
+        instances_name = [instance.name for instance in instances]
+        for instance_name in instances_name:
+            instance_config = None
+            try:
+                instance_config = load_instance_config(instance_name)
+            except ValueError:
```
Comment (antoine-de) [inline source/tyr/tyr/aggregate_places.py:178]: faudrait pas logger ici ?

File: source/tyr/tyr/aggregate_places.py (hunk 1 above)
Comment (l-vincent-l) [inline source/tyr/tyr/aggregate_places.py]: Effectivement

File: source/tyr/tyr/aggregate_places.py (hunk 2 above)
Comment (l-vincent-l) [inline source/tyr/tyr/aggregate_places.py:178]: Ouais, on peut

**Pull request merged**